src/coreComponents/physicsSolvers/fluidFlow/SinglePhaseBase.cpp

```diff
@@ -330,12 +330,6 @@ void SinglePhaseBase::initializePostInitialConditionsPreSubGroups()
                                                                MeshLevel & mesh,
                                                                arrayView1d< string const > const & regionNames )
   {
-    FieldIdentifiers fieldsToBeSync;
-    fieldsToBeSync.addElementFields( { fields::flow::pressure::key() },
-                                     regionNames );
-
-    CommunicationTools::getInstance().synchronizeFields( fieldsToBeSync, mesh, domain.getNeighbors(), false );
-
```

Comment:
This synchronization was useless because consistency of values is already guaranteed by virtue of ghosted cells being included in sets to which initial conditions are applied.

Reply:
how about other flow solvers?